ExACpLI.pm

```diff
@@ -0,0 +1,131 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2017] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+=head1 NAME
+
+ExACpLI - Add ExAC pLI to the VEP output 
+
+=head1 SYNOPSIS
+
+  mv ExACpLI.pm ~/.vep/Plugins
+  mv ExACpLI_values.txt ~/.vep/Plugins
+  perl vep.pl -i variants.vcf --plugin ExACpLI
+
+=head1 DESCRIPTION
+
+
+  A VEP plugin that adds the probabililty of being loss-of-function 
+  intolerant (pLI) to the VEP output.
+  
+  Lek et al. (2016) estimated pLI using the expectation-maximization 
+  (EM) algorithm and data from 60,706 individuals from 
+  ExAC (http://exac.broadinstitute.org/about). The closer pLI is to 1, 
+  the more likely the transcript is loss-of-function (LoF) intolerant.
```

Comment:
transcript =>gene . 
We also need something like 'Note - this score was calculated using a representative transcript and is here reported by gene.'